compiler/test/__snapshots__/basic_functionality.61c58118.0.snapshot

```diff
@@ -17,55 +17,49 @@ basic functionality › block_no_expression
  (export \"_gmain\" (func $_gmain))
  (export \"_start\" (func $_start))
  (export \"GRAIN$TABLE_SIZE\" (global $global_1))
- (func $func_0 (; has Stack IR ;) (param $0 i32) (result i32)
+ (func $f_1137 (; has Stack IR ;) (param $0 i32) (result i32)
   (i32.const 1879048190)
  )
  (func $_gmain (; has Stack IR ;) (result i32)
   (local $0 i32)
-  (return_call_indirect (type $i32_=>_i32)
-   (local.tee $0
-    (tuple.extract 0
-     (tuple.make
-      (block (result i32)
-       (i32.store
-        (local.tee $0
-         (tuple.extract 0
-          (tuple.make
-           (call $import_GRAIN$MODULE$runtime/gc_0_malloc_0
-            (global.get $import_GRAIN$MODULE$runtime/gc_0_GRAIN$EXPORT$malloc_0)
-            (i32.const 16)
-           )
-           (i32.const 0)
-          )
+  (drop
```

Comment:
In this case, Binaryen just inlined the call, so there's no `call` at all anymore. You'll notice that `f` here is still in the output even though it was inlined—that's what I'll be tackling next.